doc/user_manual/PostProcessors/Validation.tex

```diff
@@ -148,3 +149,42 @@ \subsubsection{Validation PostProcessors}
   ...
 <Simulation>
 \end{lstlisting}
+
+\paragraph{PCM}
+\textbf{PCM} evaluates the uncertainty reduction fraction when using Feature to validate Target. 
+PCM requires sub-nodes in \textit{Validation} PP including:
+
+%
+\ppType{PPDSS}{PPDSS}
+%
+
+\begin{itemize}
+  \item \xmlNode{Features}, \xmlDesc{comma separated string, required field}, specifies the names of the features.
+  \item \xmlNode{Targets}, \xmlDesc{comma separated string, required field}, contains a comma separated list of
+     targets. \nb Each target will be validated using all features listed in xml node \xmlNode{Features}. The
+    number of targets is not necessarily equal to the number of features.
+  \item \xmlNode{Measurements}, \xmlDesc{comma separated string, required field}, contains a comma separated list of
+     measurements of the features. \nb Each measurement correspond to a feature listed in xml node \xmlNode{Features}. The
+    number of measurements should be equal to the number of features and in the same order as the features listed in \xmlNode{Features}.
+\end{itemize}
+
+The output of PCM is comma separated list of strings in the format of ``pri_post_stdReduct_[targetName]", 
```

Comment:
use "\_" for "_" otherwise the manual can not be built. @huang714

Reply:
I changed to "\textunderscore" to fix this issue and it showed correctly in the built manual. Using any "_" symbol cannot be built from my side. I have also modified multiple places of PCM doc. Thx